lib/components/ImportComponentDialog2/types.ts

```diff
@@ -0,0 +1,39 @@
+export type ImportSource = "tscircuit.com" | "jlcpcb" | "kicad"
+
+export interface ComponentSearchResult {
+  id: string
```

Comment:
the fact that there's a naked `id` here indicates there's an antipattern (we never have naked ids, and the only time they're introduced is when someone is making a polymorphic foreign key)